src/specs/api.yaml

```diff
@@ -458,16 +458,27 @@ components:
           type: string
           example: GetEmbedding
           description: The name of this work -- must be `GetEmbedding`.
```

Comment:
My understanding is that the API is failing because you did not set up a pattern for the "type" string like it is done [here](https://github.com/biomage-ltd/api/blob/02d51c45bf6698128832c97558cc7b6986fe082c/src/specs/api.yaml#L489) -- for this schema, only `name` is required, and name does not have a pattern. Therefore, any and all work tasks will match with this, as well as with their respective request schemas.